config/services.yaml

```diff
@@ -15,6 +15,11 @@ services:
             - '../src/PhpParser/Builder/*'
             - '../src/HttpKernel/*'
             - '../src/ValueObject/*'
+            - '../src/Configuration/MinimalVersionChecker'
+            - '../src/Configuration/MinimalVersionChecker/*'
+
+    Rector\Core\Configuration\MinimalVersionChecker:
+        autowire: false
```

Comment:
@TomasVotruba Does it look okay? This is the only way I could disable autowiring for it.

Reply:
I'll check locally, if these can be done something about it, after merge